alot/settings/manager.py

```diff
@@ -68,15 +68,11 @@ def reload(self):
 
     def read_notmuch_config(self):
         """parse notmuch's config file from path"""
-        if self.notmuch_rc_path is not None:
```

Comment:
ahhh, I didn't look at the notmuch config path closely enough. We definately need to catch an exception there. I'll update and resend.